common/pom.xml

```diff
@@ -18,6 +18,18 @@
         <jackson.modules.version>2.8.5</jackson.modules.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
```

Comment:
This declares versions for org.springframework.cloud:* dependencies that all work nicely together.  The spring-cloud project doesn't have single-release versions like spring-core does.  There's no 1.2.3.RELEASE that applies to spring-cloud-starter-config, spring-cloud-starter-bus-amqp, etc